Nfield.Quota/QuotaFrameValidator.cs

```diff
@@ -15,9 +15,6 @@ public QuotaFrameValidator()
             // Note: This only holds for rules defined after the SAME RuleFor() call
             CascadeMode = CascadeMode.StopOnFirstFailure;
 
-            RuleFor(qf => qf.MaxOvershoot)
```

Comment:
When not having a quota structure, this still applies, right?

Reply:
It is now covered by the other rule.